clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -1078,7 +1078,7 @@ ListPartitionReassignmentsResult listPartitionReassignments(Optional<Set<TopicPa
      * @param options The options to carry removing members' information.
      * @return The MembershipChangeResult.
      */
-    MembershipChangeResult removeMemberFromConsumerGroup(String groupId, RemoveMemberFromConsumerGroupOptions options);
+    RemoveMemberFromGroupResult removeMemberFromConsumerGroup(String groupId, RemoveMemberFromConsumerGroupOptions options);
```

Comment:
Can we add `Consumer` in this name? That is, `RemoveMemberFromConsumerGroupResult`. It's quite verbose, but the other consumer group APIs include `ConsumerGroup` in the names. Eventually we may have generic group APIs supported through the AdminClient.

Reply:
Sounds good